homeassistant/components/scrape/__init__.py

```diff
@@ -58,19 +58,23 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     if config.get(DOMAIN) in [None, []]:
```

Comment:
You could use a walrus here:
```suggestion
    if not (scrape_config := config.get(DOMAIN)):
```